docs/make.jl

```diff
@@ -13,7 +13,7 @@ makedocs(
     strict = true,
     format = Documenter.HTML(prettyurls = get(ENV, "CI", nothing) == "true"),
     sitename = "JuliaInterpreter.jl",
-    authors = "Keno Fischer, Tim Holy, and others",
+    authors = "Keno Fischer, Tim Holy, Kristoffer Carlsson, and others",
```

Comment:
Woho :tada:

Reply:
TBH I don't even know if this field does anything. But clearly we haven't maintained the docs in a long time!